testdata/drive-2.img

```diff
@@ -0,0 +1 @@
+dd if=/dev/zero of=drive-2.img bs=1k count=102400
```

Comment:
Why do you have a dd command here?

Reply:
Just cp'd the image file, but will go ahead and remove the command.